tools/cockpit.spec

```diff
@@ -392,6 +392,8 @@ fi
 Summary: Cockpit user interface for storage, using Storaged
 Requires: %{name}-shell >= %{required_base}
 Requires: storaged >= 2.1.1
+Requires: python
+Requires: python-dbus
```

Comment:
Can we only do Python 2 for RHEL/CentOS, and use Python 3 on Fedora?

Reply:
Done.

(There is also `cockpit-docker` that depends on `python2`.  I guess that needs cleaning up, right?)

Reply:
No, cockpit-docker already [has](https://github.com/cockpit-project/cockpit/blob/master/tools/debian/control#L89) `Depends: python3`, and `debian.setup` does not have any python package. So that should be fine.